test/alternator/conftest.py

```diff
@@ -13,6 +13,10 @@
 import boto3
 import requests
 import re
+
+import sys
+sys.path.insert(1, sys.path[0] + '/../..')
```

Comment:
This is ugly and breaks linters / type checkers (mypy in particular).

Why do you do it? I don't think it's needed.

Reply:
I put in work to remove these in the past: ff4430d8ea1f6dfc38c43add38822fdefd773f1a

and now you're bringing them back :(

Reply:
Perhaps some subdirectories are missing __init__.py files?

Reply:
Fully agree that modifying the sys path is ugly, and inconvenient, but making it a package will require a lot of refactorings, and it's not a matter of this PR.